eng/jobs/finalize-publish.yml

```diff
@@ -8,7 +8,7 @@ jobs:
                   succeeded('Build_Linux_Arm64'),
                   succeeded('Build_Linux_Arm'),
                   succeeded('Build_Linux_x64_Alpine36'),
-                  succeeded('Build_Linux_x64_FreeBSD'),
+                  succeeded('Build_FreeBSD_x64'),
```

Comment:
I'd fix the sorting on the two changes in this file.

Reply:
Hmm, noticed the non-alpha order was in sync with `azure-pipelines.yml`. I'm going to sort that too.